packages/@okta/vuepress-site/guides/sign-into-mobile-app/configure-the-callback/android/configurescheme.md

```diff
@@ -1,5 +1,9 @@
 To do this, you must define a gradle manifest placeholder in your app's `build.gradle`:
 
-code sample
+```gradle
+android.defaultConfig.manifestPlaceholders = [
+    "appAuthRedirectScheme": "com.okta.oidc.example"
+]
+```
 
-Make sure this is consistent with the redirect URI used in `okta_app_auth_config.json`. For example, if your **Login Redirect URI** is `com.okta.example:/callback`, the **AppAuth Redirect Scheme** is `com.okta.example`.
\ No newline at end of file
+Make sure this is consistent with the redirect URI used in `okta_oidc_config.json`. For example, if your **Login Redirect URI** is `com.okta.oidc.example:/callback`, the **AppAuth Redirect Scheme** is `ccom.okta.oidc.example`.
```

Comment:
Is the `ccom` at the end of the sentence accurate?

Reply:
Fixed